keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/Validation.kt

```diff
@@ -0,0 +1,8 @@
+package com.netflix.spinnaker.keel.api
+
+import com.netflix.spinnaker.keel.api.schema.Discriminator
+
+interface Validation {
```

Comment:
Does this name work?

Reply:
Sounds good to me!

Reply:
I'd be happier if the naming was consistent with the field in the delivery config. So, either "verifyWith / Verification" or "validateWith / Validation" (I don't have a preference as to which pair).